extension/fts/src/fts_extension.cpp

```diff
@@ -1,22 +1,37 @@
 #include "fts_extension.h"
 
-#include "catalog/catalog_entry/catalog_entry_type.h"
+#include "catalog/catalog.h"
+#include "catalog/catalog_entry/index_catalog_entry.h"
+#include "catalog/fts_index_catalog_entry.h"
 #include "function/create_fts_index.h"
 #include "function/drop_fts_index.h"
-#include "function/query_fts.h"
+#include "function/query_fts_index.h"
 #include "function/stem.h"
 #include "main/client_context.h"
 #include "main/database.h"
 
 namespace kuzu {
 namespace fts_extension {
 
+static void initFTSEntries(transaction::Transaction* tx, catalog::Catalog& catalog) {
+    auto indexEntries = catalog.getIndexes()->getEntries(tx);
+    for (auto& [_, entry] : indexEntries) {
+        auto indexEntry = entry->ptrCast<catalog::IndexCatalogEntry>();
+        if (indexEntry->getIndexType() == FTSIndexCatalogEntry::TYPE_NAME) {
+            auto deserializedEntry = FTSIndexCatalogEntry::deserializeAuxInfo(indexEntry);
+            catalog.dropIndex(tx, indexEntry->getTableID(), indexEntry->getIndexName());
+            catalog.createIndex(tx, std::move(deserializedEntry));
```

Comment:
I don't think drop and re-create is a good idea here. Both operations will trigger transaction wal, which should be avoided in this case. Can you see: 1) do copy between the index entries, or 2) directly deserialize into the target catalog entry?